fmskill/observation.py

```diff
@@ -479,6 +479,20 @@ def __init__(
         if not df.index.is_unique:
             df.index = make_unique_index(df.index, offset_duplicates=offset_duplicates)
 
+        # TODO is this needed elsewhere?
```

Comment:
x_item and y_item should be parsed (and renamed) in the same way for track modelresults and track observations. But not necessarily the task of this PR.